spec/integration/group_index_spec.rb

```diff
@@ -3,25 +3,13 @@
 RSpec.describe 'Group Index', type: :feature do
   describe 'Group' do
     before(:each) do
-      @user1 = User.create! name: 'Gordon', password: 'godanah', email: 'gbuldie@gmail.com'
+      @user1 = User.create! name: 'samara', password: 'samu@2022', email: 'samara@gmail.com'
       @group = @user1.groups.create! name: 'HBO', icon: 'icon'
       visit new_user_session_path
-      fill_in('Email', with: 'gbuldie@gmail.com')
-      fill_in('Password', with: 'godanah')
+      fill_in('Email', with: 'samara@gmail.com')
+      fill_in('Password', with: 'samu@2022')
       click_button('LOG IN')
       visit root_path
     end
```

Comment:
- [ ] I will suggest you fix your tests. You still have some that are failing. The `group_index_spec.rb` is just one of them.
Please see the screenshot to see that you still have 6 failing tests that need retouching.

Test Screenshot |
:---------------------:|
![a1](https://user-images.githubusercontent.com/39523312/164947068-390d9898-88b3-4941-b8e5-1cd73e710842.png)